test/regress/cli/regress0/bv/bv_to_int_proj_601.smt2

```diff
@@ -0,0 +1,13 @@
+; EXPECT: 
+; SCRUBBER: grep -v "higher.order"
+; EXIT: 1
+; DISABLE-TESTER: dump
+(set-logic ALL)
```

Comment:
This benchmark doesn't involve higher-order. I'm not sure why this would throw the exception you introduce in this PR?